apps/core/src/hooks/useMultiGetObjects.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { ObjectId, SuiObjectDataOptions } from '@mysten/sui.js';
+import { useRpcClient } from '../api/RpcClientContext';
+import { useQuery } from '@tanstack/react-query';
+import { chunkArray } from '../utils/chunkArray';
+
+export function useMultiGetObjects(
+    ids: ObjectId[],
+    options: SuiObjectDataOptions = { showContent: true, showOwner: true }
```

Comment:
had a hell of a time trying to type this in order allow options to be passed to `useQuery` 🥲